TypeInjections/Tool/Tool.fs

```diff
@@ -21,8 +22,10 @@ module Tool =
           newFolder: string
           [<Option('p', "output", Required = false, HelpText = "path of folder to place typeCart files")>]
           outputFolder: string
-          [<Option('f', "file", Required = false, HelpText = "specify filename(s) you want to run typeCart on")>]
-          file: seq<string> }
+          [<Option('i', "ignore", Required = false, HelpText = "specify filename(s) you want to ignore")>]
+          ignore: seq<string>
+          [<Option('t', "typecartignore", Required = false, HelpText = "give path of .typecartignore file")>]
```

Comment:
Why is this not an option with the `git` verb?

Reply:
sorry I was on the fence about including `--typecartignore`. I think just having `--ignore` would be best. I will delete this